modules/telco-troubleshooting-general-check-logs.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-general-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-troubleshooting-general-check-logs_{context}"]
+= Checking pod logs
+
+Get logs from the pod.
+
+.Procedure
+
+. List the pods by running the following command:
++
+--
+[source,terminal]
+----
+$ oc get po
+----
+
+.Example output
+[source,terminal]
+----
+NAME        READY   STATUS    RESTARTS          AGE
+busybox-1   1/1     Running   168 (34m ago)     7d
+busybox-2   1/1     Running   119 (9m20s ago)   4d23h
+busybox-3   1/1     Running   168 (43m ago)     7d
+busybox-4   1/1     Running   168 (43m ago)     7d
+----
+--
+
+. Check pod log files by running the following command: 
++
+[source,terminal]
+----
+$ oc logs -n <namespace> busybox-1
+----
```

Comment:
Consider adding a line or two to the end of the procedure that directs readers to the additional resources: `For more information about reviewing pod log files, see "oc logs", "Logging", and "Inspecting pod and container logs".`